### PR TITLE
changes-monitorsdoc

### PR DESCRIPTION
**Pull request by JainM6**
## Purpose of this pull request

This pull request (PR) ...

Issue number: 

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [X ] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

**Review by nmohantyctr — APPROVED** [review]

File: docs/alerts/monitors/create-monitor.md [hunk 1]
```diff
@@ -176,17 +176,22 @@ Sumo Logic automatically resolves the incident when the resolution condition is
...
-**Alert and recovery window**
+#### Alert and recovery window
 
-This setting controls the behavior of alert/recovery for sparse/intermittent/incomplete data sources.
+This setting affects both the alert generation logic and the alert recovery logic.
 
 ![metrics alert datapoints.png](/img/monitors/minimum-datapoints.png)
 
-`Alert and recovery require a minimum of <Count> data points for “at all times” evaluation windows`
+`Alert and recovery require a minimum of <Count> data points for `at all times` evaluation windows`
```
Comment (kimsauce) [inline docs/alerts/monitors/create-monitor.md:185]:
```suggestion
`Alert and recovery require a minimum of <Count> data points for "at all times" evaluation windows`
```

This must be double quotes to match the UI (see screenshot below)
Screenshot 2023-02-07 at 12 57 42 PM

**Review by kimsauce — CHANGES_REQUESTED** [review]
Leave line 185 as-is (replacing single quotes with double quotes will break the formatting)

**Review by kimsauce — APPROVED** [review]

File: docs/alerts/monitors/create-monitor.md [hunk 2]
```diff
@@ -176,17 +176,22 @@ Sumo Logic automatically resolves the incident when the resolution condition is
...
 
 | Parameter | Description |
 |:--|:--|
-| Count | The minimum number of data points needed in alert and recovery `Time Range` for alert to trigger and resolve respectively. |  
+| Count | The minimum number of data points required within the 5-minute window to trigger an alert or recover from an alert. This means that if Sumo Logic receives fewer data points in a given window, no alert will be triggered (even if all those data points exceed the threshold).
+
+For example, you want to be alerted when the CPU usage is over 60% `at all times` within a 5-minute window. If you set the count to 3, this means that you will only get an alert if you have at least 3 data points showing CPU usage above 60% within that 5-minute window. If you only have 2 data points, even if both of them show CPU usage above 60%, you won't get an alert.
+:::note
+This setting only works when you choose `at all times within` as the type of occurrence for the alert.  
+:::  |
```
Comment (kimsauce) [inline docs/alerts/monitors/create-monitor.md:194]:
```suggestion
:::
```

File: docs/alerts/monitors/create-monitor.md [hunk 3]
```diff
@@ -176,17 +176,22 @@ Sumo Logic automatically resolves the incident when the resolution condition is
...
 
 ![metrics alert datapoints.png](/img/monitors/minimum-datapoints.png)
 
-`Alert and recovery require a minimum of <Count> data points for “at all times” evaluation windows`
+`Alert and recovery require a minimum of <Count> data points for "at all times" evaluation windows`
 
 | Parameter | Description |
 |:--|:--|
-| Count | The minimum number of data points needed in alert and recovery `Time Range` for alert to trigger and resolve respectively. |  
+| Count | The minimum number of data points required within the 5-minute window to trigger an alert or recover from an alert. This means that if Sumo Logic receives fewer data points in a given window, no alert will be triggered (even if all those data points exceed the threshold).
```
Comment (tarunk2) [inline docs/alerts/monitors/create-monitor.md:189]: The window might not always be 5 mins. `The minimum number of data points required within the configured window.

Comment (kimsauce) [inline docs/alerts/monitors/create-monitor.md:189]:
```suggestion
| Count | The minimum number of data points required within the configured window to trigger an alert or recover from an alert. This means that if Sumo Logic receives fewer data points in a given window, no alert will be triggered (even if all those data points exceed the threshold).
```

**Pull request merged**